src/plugins/visualizations/public/vis.ts

```diff
@@ -17,6 +17,7 @@
  */
 
 import { isFunction, defaults, cloneDeep } from 'lodash';
+import { produce } from 'immer';
```

Comment:
I wonder if this has any bundle size impact actually. At the end is used just to update two props.

Reply:
Yeah, good point. I'll update it to just use Javascript.